### DISCUSSION TITLE
How to open .slnf files under macOS in VS?

### DISCUSSION THREAD

**Discussion by stanimiryakimov** [dotnet/aspnetcore#24003] [General]
Hey guys, I follow the [Build from source guide](https://github.com/dotnet/aspnetcore/blob/master/docs/BuildFromSource.md) I've just done restore.sh and now I wonder how can I open the .slnf files under macOS using Visual Studio? Double click on a .slnf results in opening the solution file as plain text using Visual Studio and not the actual filtered set of projects.

**Comment by Tratcher** [comment]
@Pilchie ?

**Reply by Pilchie** [reply]
I'm following up.  This may be a problem with the move to slnf files ☹️

**Reply by stanimiryakimov** [reply]
Hi @Pilchie  and @Tratcher . Same question in [Developer Community](https://developercommunity.visualstudio.com/content/problem/1117064/cant-open-slnf-file-under-macos.html). Under Windows, does .slnf files work the same way as .sln - just double click and loads all projects associated with the solution?

**Reply by Pilchie** [reply]
Yes, slnf files allow you to load a subset of all of the projects in a .sln.

**Comment by Pilchie** [comment]
I've been following up with the team.  Visual Studio for Mac *doesn't* support slnf files right now, but the team's current plan is to add support for opening them in the 8.9 release.
